### ISSUE TITLE
Typo in error message: pymethods

### ISSUE THREAD

**Issue by MrWinstead** [PyO3/pyo3#436]
## 🐛 Bug Reports

When reporting a bug, please provide the following information. If this is not a bug report you can just discard this template.

### 🌍 Environment

 - Your operating system and version:
```
17:37 $ cat /etc/os-release 
NAME=Fedora
VERSION="29 (Twenty Nine)"
ID=fedora
VERSION_ID=29
```
 - Your python version:
```
19:02 $ python3 --version
Python 3.7.2
```
 - How did you install python (e.g. apt or pyenv)? Did you use a virtualenv?:
python3 is installed via dnf
 - Your rust version (`rustc --version`):
```
19:02 $ rustc --version
rustc 1.35.0-nightly (2975a3c4b 2019-04-15)
```
 - Are you using the latest pyo3 version? Have you tried using latest master (replace `version = "0.x.y"` with `git = "https://github.com/PyO3/pyo3")?` 
This shouldn't matter for the error message typo

### 💥 Reproducing

Using the following invokes the error that one can't use `#[pymethods]` with lifetimes:
```
#[pyclass]
pub struct Kata<'a> {
    project: &'a PyObject,
    angr_module: &'a PyModule,
}

#[pymethods]
impl<'a> Kata<'a> {
    #[new]
    pub fn new(obj: &PyRawObject, project: &PyObject) {
        let gil = Python::acquire_gil();
        let py = gil.python();

        let angr_module = py.import("angr");
        if !angr_module.is_ok() {
            let mut err_msg = PyString::new(py, "could not import angr").to_object(py);
            unsafe {
                PyErr_SetImportError(err_msg.as_ptr(), null(), null())
            }
            return
        }

        obj.init(Kata {
            project,
            angr_module: angr_module.unwrap(),
        });
    }
}
```

That's why this ticket exists. In the output message, there's a typo (`ve` instead of `be`):
```
...
/home/mike/.cargo/bin/cargo +nightly build --color=always
   Compiling dojo v0.1.0 (/home/mike/repos/github.com/mrwinstead/dojo)
error: #[pymethods] can not ve used with lifetime parameters or generics
  --> src/dojo/kata.rs:14:5
   |
14 | impl<'a> Kata<'a> {
   |  
...
```

I'll file the PR shortly.

**Comment by MrWinstead** [comment]
I just re-tried with master with the same error message.

**Comment by kngwyu** [comment]
Thanks!
Looks the issue is here https://github.com/PyO3/pyo3/blob/master/pyo3-derive-backend/src/pyimpl.rs#L16

**Comment by konstin** [comment]
As far as I know there's no way to make lifetimes sound with python objects, i.e. we can't enforce that the referenced objects actually outlive our object after we moved it into a python object.

**Comment by kngwyu** [comment]
@konstin 
This issue only points out typo(`ve` should be `be`).

**Comment by kngwyu** [comment]
Fixed in #440

**Comment by konstin** [comment]
Ohh, I'm sorry. Thanks for the fix though!
